frontend/controller/actions/group.js

```diff
@@ -606,6 +618,10 @@ export default (sbp('sbp/selectors/register', {
 
     const message = await sbp('gi.actions/chatroom/create', {
       data: {
+        attributes: {
+          ...params.data?.attributes,
+          adminIDs: [contractState.groupOwnerID]
+        },
```

Comment:
By placing `adminIDs` after `...params` it is now impossible to call `'gi.actions/group/addChatRoom'` and specify `adminIDs`. Perhaps the order should be reversed?

Reply:
That was on purpose, as I though the admin should always be the group owner.

Reply:
Put admin first so that it can be overridden.